app/components/team/Index.js

```diff
@@ -283,7 +282,7 @@ class TeamIndex extends React.PureComponent {
                 theme="default"
                 outline={true}
               >
-                Create a Team
+                New Team
```

Comment:
Dang, I did change that in one of the PRs I swear… but it appears to have been lost. Boooo